test/verify/check-system-realms

```diff
@@ -437,7 +437,7 @@ class CommonTests:
         do_test(alice_user_pass, ['HTTP/1.1 200 OK', '"csrf-token"'], session_leader='cockpit-session', retry=True)
 
         # enable cert based auth
-        m.write("/etc/cockpit/cockpit.conf", '[WebService]\nClientCertAuthentication = true\n', append=True)
+        m.write("/etc/cockpit/cockpit.conf", 'ClientCertAuthentication = true\n', append=True)
```

Comment:
This change is ugly. It relies on the previous content of cockpit.conf.

The fundamental problem is that we can not repeat sections in the file, so this will be rejected:

```
[WebService]
foo = 12

[WebService]
bar = 8
```

I think this could be allowed.

Reply:
That's the bug:  on hosts where  multihost is still enabled, it's missing the `[WebService]` header. So this needs to become another if/then.

Reply:
Fixed.